src/config/nunjucks.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     const nunjucksEnv = nunjucks.configure(viewsPath, {
       autoescape: true,
       express: app,
-      noCache: true,
+      noCache: !process.env.TEMPLATE_CACHING,
```

Comment:
I think you probably need to add this to
- `config.js` (so all the config can be found in one place)
- `template.yaml` (so it's set appropriately in the deployed environments)

Reply:
Ah, ok I didn't realise environment variables could be set in template.yaml, will make edits 👍